### PR TITLE
Followup - WINTERAPI: implement DELETE + add target name in POST payload

### PR DESCRIPTION
**Pull request by Theodlz**
- add a delete method to the WINTERAPI, that grabs the schedule name from the POST response (from facility transactions) and calls the `/too/delete` endpoint.
- add a method to grab said schedule name from the POST response
- send the target_name (obj_id) to WINTER when submitting a request (max 30 chars).

### CODE REVIEW

**Comment by Theodlz** [comment]
@robertdstein just so you know that exists

**Comment by mcoughlin** [comment]
@Theodlz just confirming that you see the failed tests.

File: skyportal/facility_apis/winter.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
+import json
 import requests
+import urllib
 from requests.auth import HTTPBasicAuth
 from datetime import datetime, timedelta
 from astropy.time import Time
-import urllib
 
-from . import FollowUpAPI
+from . import FollowUpAPI, FacilityTransaction
```
Comment (mcoughlin) [inline skyportal/facility_apis/winter.py:8]: This likely needs to be imported inside.

File: skyportal/facility_apis/winter.py [hunk 2]
```diff
@@ -89,6 +91,28 @@ def _build_payload(self, request):
 
         return target
 
+    def schedule_name(request):
+        content = request.transactions[0].response["content"]
+        content = json.loads(content)
+        # the WINTERAPI POST response has a "msg" key,
+        # that contains Schedule name is <schedule_name> if successful
+        if "msg" in content:
+            # just to that we are not sensible to the exact format of the message
```
Comment (mcoughlin) [inline skyportal/facility_apis/winter.py:100]: sensible is a weird word.

**Review by mcoughlin — APPROVED** [review]
Probably just need to move that import.

File: skyportal/facility_apis/winter.py (hunk 2 above)
Comment (Theodlz) [reply skyportal/facility_apis/winter.py:100]: @mcoughlin that's the french in me sorryyyyy. Will fix ASAP. I also fixed the import already

**Pull request merged**